src/views/Incorporation/IncorporationReviewConfirm.vue

```diff
@@ -220,6 +220,29 @@
       </v-card>
     </section>
 
+    <!-- Document ID Component for Staff only -->
+    <section
+      v-if="IsAuthorized(AuthorizedActions.STAFF_PAYMENT) && !isEntityCoop"
```

Comment:
Please use `IsAuthorized(AuthorizedActions.DOCUMENT_RECORDS)` as in the previous component.